src/applications/vaos/components/calendar/CalendarOptionsSlots.jsx

```diff
@@ -95,10 +94,7 @@ export default function CalendarOptionsSlots({
         const checked = selectedDates.some(
           selectedDate => selectedDate === slot.start,
         );
-        let time = moment(slot.start);
-        if (slot.start.endsWith('Z') && timezone) {
-          time = time.tz(timezone);
-        }
```

Comment:
removing this condition fixes the slot times being displayed on the calendar radio option

Reply:
@jbalboni, I removed a condition in the `calendarOptionsSlots` to have the selected time from the calendar match what's displayed on the `Review appointment` page. 

Would you please review the code to see if I corrected the right section of code?

Reply:
I don't think we want to remove this condition, I think we may need to update the review page with the same logic.

Based on my testing, the 2021-08-06T15:20:00Z slot is being displayed as 11:20am after this change, but that's ET and it should be displaying in MT time, which is 9:20.

Reply:
@jbalboni Thanks for explaining the time difference. I'll revert this file.